### ISSUE TITLE
[IDEA] Node Filter Operators and Backtags

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#8252]
A node filter operator is one returns one form of nodes as below
- link
- backlink
- transclude
- backtransclude
- tag
- tagging (backtag)

These filters can be used to explorer nodes and construct a bidirectional map of information (ZettelKasten note taking).

**Idea 1**
Is it possible to add an alias for tagging? For **consistency** reason I propose the `backtags`. This way both tagging and backtags share the same code and acts as aliases for eachother.

The set of node filter operators then will be

* `tags`
* `backtags`
* `links`
* `backlinks`
* `transcludes`
* `backtranscludes`

**Idea 1**
Is it worth to add this terminology to official documentation:
- what is a node in bidirectional note taking/information mapping?
 - what is node filter operators?
 - and add a new documentation tag `Node Filter Operators`

**Comment by kookma** [comment]
@sobjornstad, @linonetwo 
As you both works with other note talking apps, is backtags a correct word here? or other apps use better word?

**Comment by linonetwo** [comment]
`backtags` is easy to remember. But we already have `tagging` Operator.

**Comment by Jermolene** [comment]
Hi @kookma we have found that the "tag" and "tagging" operators are confusing for users, but I'm not sure that "backtags" would be any better. I do appreciate the symmetry with the other "back*" operators, and perhaps that is worth pointing out in the docs.

**Comment by kookma** [comment]
> `backtags` is easy to remember. But we already have `tagging` Operator.

Can't we make backtags as alias for tagging? I mean to keep tagging but allow backtags refer to tagging?

**Comment by sobjornstad** [comment]
@kookma RemNote calls the items tagged “instances” of the tag, perhaps too technically. (In RemNote tags can also assign a type to the thing they tag, so the OOP metaphor makes sense sense) I don't think this would be helpful in TW.

Otherwise I've only heard “tagging” or “tagged” here. I do find the `tagging` operating confusing – sometimes I have to look it up to remind myself of the difference between `tagging` and `tags`.

“Backtags” sounds kind of odd to me, but I like the idea of having it as an alias.

**Comment by linonetwo** [comment]
Well, why backlinks is OK but backtags is odd? But I'm not sure if other app also use backlinks as a term.

**Comment by sobjornstad** [comment]
Yeah I think that's why – “backlinks” is a well-established term used by almost every app that has this feature.

**Comment by Jermolene** [comment]
> Can't we make backtags as alias for tagging? I mean to keep tagging but allow backtags refer to tagging?

I would caution against doing so because it adds significant cognitive overhead for users to have to know about both terms, and it would be necessary to know both terms if one was reading other peoples code.

**Comment by kookma** [comment]
Thank you all.
Based on @Jermolene suggestion I add some explanation to official docs.
